datahub-web-react/src/app/entity/chart/ChartEntity.tsx

```diff
@@ -10,6 +11,25 @@ import ChartProfile from './profile/ChartProfile';
 export class ChartEntity implements Entity<Chart> {
     type: EntityType = EntityType.Chart;
 
+    icon = (fontSize: number, styleType: IconStyleType) => {
+        if (styleType === IconStyleType.TAB_VIEW) {
+            return <LineChartOutlined style={{ fontSize }} />;
+        }
+
+        if (styleType === IconStyleType.HIGHLIGHT) {
+            return <LineChartOutlined style={{ fontSize, color: 'rgb(144 163 236)' }} />;
+        }
+
+        return (
+            <LineChartOutlined
+                style={{
+                    fontSize,
+                    color: '#BFBFBF',
+                }}
+            />
+        );
+    };
```

Comment:
is it possible to let the tab / highlight / etc manage the color instead? otherwise if someone forks and adds their own entity, if these colors ever change their entity won't pick that up.

Reply:
So here's the thing: We often render these icons in a list (eg group of Datasets, group of Users, etc). If we want each to have a consistent "main color" there's no way for the caller to know what that should be. But you're right the alternative is allowing the color to pass a color themselves

Reply:
Maybe then just factor this out into a shared const so new entities can reference the const to be sure they'll be updated if any refactor occurs?